Environmental/new_crawlers/Weather forecast/anta_weather_forecast/producer.py

```diff
@@ -0,0 +1,48 @@
+import os
+import json
+import requests
+import datetime
+from kafka import KafkaProducer
+from dotenv import load_dotenv
+
+ANTA_LAT = 36.963974
+ANTA_LON = 30.727117
+ANTA_RESOLUTION = "6km"
+FROM_DATE = datetime.datetime.now().strftime("%Y-%m-%d")
+TO_DATE = (datetime.datetime.now() + datetime.timedelta(days=7)).strftime("%Y-%m-%d")  # TO_DATE = FROM_DATE + 7 days
+VARIABLES = ["temperature2m", "rh2m", "windspeed10m", "precipitation"]
+TIMEZONE = "Europe/Athens"
```

Comment:
These seem to be specific to `get_seven_day_forecast` method, maybe it is better if they are local variables there